### PR TITLE
Mul128

### PR DESCRIPTION
**Pull request by chwilko**
Added functionally Mul128 : uint64, uint64 -> bytes

### CODE REVIEW

File: pytealext/mul128.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class Mul128(Expr):
+    """
+    Mul128 calculates the expression m1 * m2, where m1 and m2 are TealType.uint64.
+    The result of this operation is a bytes.
+    """
+
+    def __init__(self, m1: Expr, m2: Expr):
+        """
+        Args:
+            ceiling: Calculate the m1 * m2
```
Comment (StylishTriangles) [inline pytealext/mul128.py:26]: invalid docstring

File: pytealext/mul128.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+from pyteal.types import TealType, require_type
+
+from .assemble import assemble_steps
+
+
+class Mul128(Expr):
+    """
+    Mul128 calculates the expression m1 * m2, where m1 and m2 are TealType.uint64.
+    The result of this operation is a bytes.
```
Comment (StylishTriangles) [inline pytealext/mul128.py:20]:
```suggestion
    The result of this operation is an integer, encoded big-endian as bytes.
```

File: pytealext/mul128.py (hunk 1 above)
Comment (chwilko) [reply pytealext/mul128.py:26]: similar muldiv64 ;D

File: tests/mul128_test.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+from hypothesis import given, settings, example
+from hypothesis.strategies import integers
```
Comment (StylishTriangles) [inline tests/mul128_test.py:4]:
```suggestion
from hypothesis import strategies as st
```
to match convention

File: tests/mul128_test.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    Int,
+    ScratchVar,
+    TealType,
+    Mode,
+    Seq,
+)
+
+THIS_DIR = os.path.dirname(os.path.realpath(__file__))
+THIS_DIR = os.path.dirname(os.path.realpath(THIS_DIR))
+sys.path.insert(1, THIS_DIR)
```
Comment (StylishTriangles) [inline tests/mul128_test.py:16]: should be removed

Comment (chwilko) [reply tests/mul128_test.py:16]: of course

File: tests/mul128_test.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+from pytealext.mul128 import Mul128
+
+from tests.helpers import compile_and_run
+
+
+VERSION = 6
+MAX_UINT64 = 2 ** 64 - 1
+
+
+
```
Comment (StylishTriangles) [inline tests/mul128_test.py:23]: There shouldn't be 3 empty lines -> Format the file with `black -l 120`

**Review by StylishTriangles — APPROVED** [review]
LGTM

**Pull request merged**